pkg/ui/src/views/reports/containers/debug/debug.styl

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018 The Cockroach Authors.
```

Comment:
policy question: I have been naming these `index.styl`, since we do `index.tsx`. I do kind of like naming files after the actual component though :P What do you think?

Reply:
I don't know.  Maybe all our styles should just be inline.